plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -76,7 +88,22 @@ func (a *Elasticsearch) Connect() error {
 
 	var clientOptions []elastic.ClientOptionFunc
 
+	tlsCfg, err := internal.GetTLSConfig(a.SSLCert, a.SSLKey, a.SSLCA, a.InsecureSkipVerify)
+	if err != nil {
+		return err
+	}
+	tr := &http.Transport{
+		ResponseHeaderTimeout: a.Timeout.Duration,
```

Comment:
Let's remove the header timeout, I think we usually don't usually need this but it keeps getting copied around from other plugins...